### ISSUE TITLE
Delete(expression) throws exception: Failed to process query: DELETE FROM "....": Incorrect syntax

### ISSUE THREAD

**Issue by habib535** [Starcounter/Starcounter.Linq#63]
Query works fine with Db.SQL() but throws an exception when executed through DbLinq.
Starcounter version: `Version=2.4.990.180` custom mapping installer

Clone https://github.com/Starcounter/HeadsRetailConnector and execute `http://localhost:8080/HeadsRetailConnector/TestQueue`

![image](https://user-images.githubusercontent.com/2991381/47498433-91887f00-d85d-11e8-96d6-b3a00b2c1eb8.png)

```
Starcounter.SqlException: 'Failed to process query: DELETE FROM "HeadsRetailConnector"."Domain"."Entities"."OrderQueueItem" WHERE ((("OrderNo" = ?) AND (O."ObjectNo" <= ?))): Incorrect syntax. '
```

The same line works fine with `Db.SQL($"DELETE FROM {nameof(OrderQueueItem)} WHERE OrderNo = ? AND ObjectNo <= ?", item.Item2, item.Item1);`

**Comment by un-tone** [comment]
Fixed, but the package is not published. Waiting for merging (or cancelling) the other opened PR.

**Comment by un-tone** [comment]
The package is published [2.3.0](https://github.com/Starcounter/Starcounter.Linq/releases/tag/2.3.0)
